### ISSUE TITLE
Bug: v4.2-beta3 - Fatal error: Class Jetpack_Sync_Defaults not found

### ISSUE THREAD

**Issue by chuckreynolds** [Automattic/jetpack#4641]
#### Steps to reproduce the issue
wp-cli: `wp core check-update`

```
PHP Fatal error: Class 'Jetpack_Sync_Defaults' not found in /wp-content/plugins/jetpack/sync/class.jetpack-sync-wp-replicastore.php on line 147
PHP Stack trace:
PHP   1. {main}() /wp-cron.php:0
PHP   2. do_action_ref_array() /wp-cron.php:117
PHP   3. call_user_func_array:{/wp-includes/plugin.php:601}() /wp-includes/plugin.php:601
PHP   4. Jetpack_Heartbeat->cron_exec() /wp-includes/plugin.php:601
PHP   5. Jetpack_Heartbeat::generate_stats_array() /wp-content/plugins/jetpack/class.jetpack-heartbeat.php:86
PHP   6. Jetpack_Sync_WP_Replicastore->checksum_all() /wp-content/plugins/jetpack/class.jetpack-heartbeat.php:144
PHP   7. Jetpack_Sync_WP_Replicastore->posts_checksum() /wp-content/plugins/jetpack/sync/class.jetpack-sync-wp-replicastore.php:574

```

jetpack v4.2-beta3

**Comment by jeherve** [comment]
Could you try to update to Beta 4? This might have been solved in 2078a3813e389def42907aac7ceab04c8a59f98c.

**Comment by lezama** [comment]
Closing this as it has been resolved by the commit pointed above, thanks for the report!
